.github/labeler-module.yml

```diff
@@ -78,3 +78,8 @@ module:tree:
 
 module:utils:
 - sklearn/utils/**/*
+
+cython:
+- sklearn/**/*.pyx
+- sklearn/**/*.pxd
+- sklearn/**/*.pxi
```

Comment:
We also have Tempita files like `sklearn/linear_model/_sag_fast.pyx.tp` which are "Cython" files.